src/ui/components/ui/Address/Tooltip.tsx

```diff
@@ -0,0 +1,44 @@
+import * as styles from './Tooltip.module.css';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Tooltip } from '../tooltip';
+import { Modal, Copy, Ellipsis } from '..';
+
+export interface Props {
+  className?: string;
+  address: string;
+}
+
+export function AddressTooltip({ className, address }: Props) {
+  const { t } = useTranslation();
+
+  const [showCopyNotification, setShowCopyNotification] = React.useState(false);
+
+  return (
+    <>
+      <Tooltip content={address}>
+        {props => (
+          <Copy
+            text={address}
+            onCopy={() => {
+              setShowCopyNotification(true);
+              setTimeout(() => setShowCopyNotification(false), 1000);
+            }}
+          >
+            <div className={className} {...props}>
+              <Ellipsis className={styles.address} text={address} size={8} />
```

Comment:
May be also display the text "Click to copy to clipboard" like in the `TransactionWallet` component?

https://github.com/wavesplatform/waves-keeper/blob/master/src/ui/components/wallets/TransactionWallet.tsx#L65:L66